CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - UI: Add country selection step after document selection
 - Internal: The release script and the `release/RELEASE_GUIDELINE.md` file now include the information needed to update the `MIGRATION.md` file.
 
+### Changed
+
+- Public: Changed resolution constraints for live document captures from `720` to `1080`.
```

Comment:
I kind of feel like this is more an internal change than public, since while it is a follow up from the public fix that was released it's not documented publicly what the resolution constraint change in that fix or the initial implementation was.